File: ChatApp.API/ChatApp.Domain/Entities/Message.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ChatApp.Domain.Entities
{
    [Table("Message")]
    public class Message
    {
        [Key]
        public int MessageId { get; set; }
        public string MessageContent { get; set; }
        public int MessageBy { get; set; }
        public DateTime MessageDateAndTime { get; set; }
        public bool IsDeleted { get; set; }
        public int ChatRoomId { get; set; }
    }
}

File: ChatApp.Client/src/app/models/message.ts
export class Message {
  messageId: number;
  messageContent: string;
  messageBy: string;
  messageDateAndTime: Date;
  chatRoomId: number;
  userId: number;
}

export class MessagesByChatRoomIdResponseModel{
    messagesDTOs: Message[];
}

export class MessagesRequestModel{
  messageContent: string;
  messageBy: number;
  messageDateAndTime: Date;
  chatRoomId: number;
}
